src/main/java/pages/actions/CdfGcsActions.java

```diff
@@ -20,6 +22,9 @@ public static void enterReferenceName()  {
     }
 
     public static void enterProjectId()  {
+        for (int i = 0; i <=100; i++) {
```

Comment:
Why are we doing it?

Reply:
I think it's better just to call `clear()`

Reply:
I have tried with that but the issue is clear doesn't work on reactive or intelligent fields. This is the workaround via keyboard. It is also logged as a web driver issue. Please follow the link - https://github.com/SeleniumHQ/selenium/issues/6741

Reply:
Other work around-WebElement ele = webDriver.findElement(By.xpath(".//input[@name='username']"));
      new Actions(webDriver).click(ele)
        .pause(200).keyDown(Keys.CONTROL).sendKeys("a").keyUp(Keys.CONTROL)
        .pause(200).sendKeys(Keys.BACK_SPACE)
        .pause(200).sendKeys("KOOKOK").perform();

I believe for look if fast to execute that pausing for performing

Reply:
Could you please extract it into the utility class and method and add a javadoc with the link above? This would explain it.
I may make sense to pass new value as well, this way its would look similar to `util.ReplaceElementValue(element, 'value')`

Reply:
Added method replaceElementvalue and changed the scope of Junit dependency to text. Project value is hardcoded as there is separate PR. Once you will approve this will raise that change in the other PR

Reply:
java doc added